.gitignore

```diff
@@ -21,3 +22,4 @@ venv/
 *.sqlite3
 
 flask_example/config.py
+fastapi_example/.env
```

Comment:
you can add this in your `fastapi_example` folder:

```
fastapi_example/.gitignore
```

Reply:
I place it here just like the `flask_example/config.py`

Reply:
oh, sorry, this is correct.